serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/servicerecord/ServiceRecordFactory.java

```diff
@@ -134,58 +132,52 @@ public ServiceRecordFactory(ServiceregistryProperties properties,
         return Optional.of(serviceRecord);
     }
 
-    @SuppressWarnings("squid:S1166") // Suppress Sonar due to no rethrow/log from certificate ex.
+    public Optional<ServiceRecord> createEduErrorRecord(String orgnr) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "orgnr". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1172)